src/test/java/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,96 @@
+package dao;
+
+import db.Storage;
+import java.util.AbstractMap;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitDaoImplTest {
+    private static FruitDao fruitDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.fruitStock.put("apple", 120);
+        Storage.fruitStock.put("banana", 65);
+    }
+
+    @Test
+    void getBalance_validFruitName_isOk() {
+        String expectedName = "apple";
+        int expectedQuantity = 120;
+        String actualName = Storage.fruitStock.keySet().stream()
+                .filter(k -> k.equals(expectedName))
+                .findFirst()
+                .get();
```

Comment:
you just can get from map

Reply:
If you can, please show me because I didn't find the way to do it.